app/graphql/concerns/validate_course_export.rb

```diff
@@ -0,0 +1,33 @@
+module ValidateCourseExport
+  extend ActiveSupport::Concern
+
+  class RequireValidCourse < GraphQL::Schema::Validator
+    def validate(_object, _context, value)
+      course = Course.find_by(id: value[:course_id])
+      return if course.present?
```

Comment:
The formatter needs to be checked. There needs to be a line break surrounding guard clauses.